Update docs for Workspace IA changes (#1360)
astronomer/docs: astro/add-user.md

1. In the Cloud UI, select a Workspace.

> 2. Click **Workspace Settings.**

Reviewer: ```suggestion
2. Click **Workspace Settings**.
```

3. In the **Access** tab, click **Add Member**.

    ![Workspace user invite button](/img/docs/add-workspace-member.png)

> 4. Enter the user's email.

Reviewer: ```suggestion
4. Select the user's name and email address in the **Organization Member** list.

5. Select a role for the user and then click **Add Member**.
```

1. In the Cloud UI, select a Workspace.

2. Click **Workspace Settings**.

> 3. In the **Access** tab, click **Edit** next to the user's information.

Reviewer: ```suggestion
3. In the **Access** tab, click **Edit** next to the user you want to remove.
```

> 4. Click **Remove member**.

Reviewer: ```suggestion
4. Click **Remove member**.
5. Click **Yes, Continue** in the confirmation dialog.
```
![image](https://user-images.githubusercontent.com/104205257/196467967-e565135d-abf9-42ca-944b-49d471d54bc6.png)
